mysql-test/std_data/sst_garbd_test.sh

```diff
@@ -27,4 +28,8 @@ echo 2
 socat TCP-LISTEN:$1,reuseaddr - > /dev/null
 echo 3
 
+sleep 5
```

Comment:
This is some stability hack, or just to make SST process longer?

Reply:
This is just for testing: this way the MTR test always triggers the terminate code path.